### PR TITLE
[skip ci] DNM: contrib/build-push-ceph-container-imgs-arm64.sh: Really fixing arch

### PR DESCRIPTION
**Pull request by djgalloway**
Signed-off-by: David Galloway <dgallowa@redhat.com>

### CODE REVIEW

**Comment by djgalloway** [comment]
Testing this here: https://2.jenkins.ceph.com/job/ceph-container-build-push-imgs-arm64/18/console

File: contrib/build-push-ceph-container-imgs-arm64.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ function build_and_push_latest_bis {
   # latest-bis is needed by ceph-ansible so it can test the restart handlers on an image ID change
   # rebuild latest again to get a different image ID
   make RELEASE="$RELEASE"-bis FLAVORS="${CEPH_RELEASES[-1]},centos-arm64,7" BASE_IMAGE=centos:7 build
-  docker tag ceph/daemon:"$BRANCH"-bis-"${CEPH_RELEASES[-1]}"-centos-7-arm64 ceph/daemon:latest-bis
+  docker tag ceph/daemon:"$BRANCH"-bis-"${CEPH_RELEASES[-1]}"-centos-arm64-7-${HOST_ARCH} ceph/daemon:latest-bis
```
Comment (leseb) [inline contrib/build-push-ceph-container-imgs-arm64.sh]: Actually, the whole function is not needed. Please remove `build_and_push_latest_bis` completely.
Thanks!

Comment (leseb) [reply contrib/build-push-ceph-container-imgs-arm64.sh]: Could please also fix line 40 with:

` source "$SCRIPT_DIR"/build-push-ceph-container-imgs.sh`

So the bash linter will be happy.

Comment (leseb) [reply contrib/build-push-ceph-container-imgs-arm64.sh]: Thanks for looking into this, let's remove the function, that should solve our problem.

Comment (leseb) [reply contrib/build-push-ceph-container-imgs-arm64.sh]: Don't give up 😀

Comment (djgalloway) [reply contrib/build-push-ceph-container-imgs-arm64.sh]: https://2.jenkins.ceph.com/job/ceph-container-build-push-imgs-arm64/22/console

Same result.  Image name is `master-52672ea-luminous-centos-arm64-7-aarch64` but function expects `master-52672ea-luminous-centos-7-aarch64`.

Comment (leseb) [reply contrib/build-push-ceph-container-imgs-arm64.sh]: Alright, don't bother with this anymore, I'll look into this with @ErwanAliasr1 next week once he returns from PTO. We might need to fix the build engine.
CI pipeline is more important 😀 .

Don't close the PR though.
Thanks.

**Comment by leseb** [comment]
Thanks for attempting a fix @djgalloway I appreciate it, I just got the thing working I guess here https://github.com/ceph/ceph-container/pull/1053 so I'm closing this.

**Pull request closed without merging**